tasks/main.yml

```diff
@@ -18,3 +18,29 @@
     name: "{{ item }}"
     global: yes
   with_items: npm_packages
+
+- name: Clear and clean npm cache
+  shell: "npm cache clean -f"
+
+- name: Install n to manage versions of node
+  shell: "npm install -g n"
```

Comment:
since we have npm module, please don't use shell

http://docs.ansible.com/ansible/npm_module.html